src/components/TemplateDetails/Summary.vue

```diff
@@ -0,0 +1,112 @@
+<template>
+  <div class="whatsapp_template_summary">
+    <div class="whatsapp_template_summary__title">
+      <div class="whatsapp_template_summary__title__main">
+        {{ this.$t('WhatsApp.template_details.summary.performace_overview') }}
```

Comment:
The `this` keyword should not be used inside templates. There are multiple occurrences in this file.